CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Fixed
+
+- Fixed bug where the k8sclusterreceiver emits a few Kubernetes cpu improperly (#419)
+  - The Splunk Otel Collector v0.47.0 added a feature gate to enable a bug fix that makes the k8sclusterreceiver emit
+  a few Kubernetes cpu metrics differently to properly adhere to OpenTelemetry specifications.
+  - Affected Metrics: k8s.container.cpu_request, k8s.container.cpu_limit, k8s.node.allocatable_cpu
+  - WARNING - Upcoming Breaking Change: If you have monitoring involving the affected metrics, then please read the
+  documentation for the
+  [receiver.k8sclusterreceiver.reportCpuMetricsAsDouble](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#highlighted-feature-gates)
+  feature gate for upcoming changes.
```

Comment:
We usually add instructions for breaking changes in this doc https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md that is referenced from changelog when needed.

In the upgrade guidelines we need to describe it step by step, like: 1. check if you use the metrics, 2. update dashboards if needed, 3. enable feature gate in for the chart.

Reply:
Got it. Good to know. Done.